vdp/pipeline/v1beta/format.proto

```diff
@@ -0,0 +1,119 @@
+syntax = "proto3";
+
+package vdp.pipeline.v1beta;
+
+import "google/protobuf/struct.proto";
+
+// JsonValue represents a JSON value.
+message JsonValue {
```

Comment:
```suggestion
// JSONValue represents a JSON value.
message JSONValue {
```

I think we should follow this guide for [initialisms](https://google.github.io/styleguide/go/decisions#initialisms).